tests/csm/rest/test_capacity_usage.py

```diff
@@ -1106,6 +1111,42 @@ def test_39923(self):
         self.log.info("[END] completed")
         self.deploy = True
 
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-45679')
+    def test_45679(self):
+        """
+        Verify GET cluster topology in degraded mode
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("[START] Failure loop")
+        for failure_cnt in range(1, self.kvalue + 1):
+            self.log.info("Starting failure loop for iteration %s ", failure_cnt)
+            self.log.info("Step 1: Send Get cluster topology")
+            resp = self.csm_obj.get_system_topology()
```

Comment:
This is duplicate code. Can u remove this call from here as it is already called line 1146

Reply:
As per test design, get topology is supposed to happen before and after node shutdown

Reply:
That will still happen. This piece of code is duplicate.